neqo-transport/src/lib.rs

```diff
@@ -31,15 +31,19 @@ pub mod tparams;
 mod tracking;
 
 pub use self::cid::{ConnectionId, ConnectionIdManager};
-pub use self::connection::{Connection, FixedConnectionIdManager, Output, State, ZeroRttState};
+pub use self::connection::{
+    Connection, DraftVersion, FixedConnectionIdManager, Output, State, ZeroRttState,
+};
 pub use self::events::{ConnectionEvent, ConnectionEvents};
 pub use self::frame::CloseError;
 pub use self::frame::StreamType;
 pub use self::stream_id::StreamId;
 
 /// The supported version of the QUIC protocol.
 pub type Version = u32;
```

Comment:
I think that we can remove this from the public API.  We can use the enum.

Reply:
yep!